pyproject.toml

```diff
@@ -12,7 +12,6 @@ filterwarnings = []
 
 [tool.black]
 line-length = 88
-skip_magic_trailing_comma = true
```

Comment:
I would keep this unless you have a strong argument to remove it. The reason to keep it is that it ignores trailing commas when deciding on whether to reformat a multi-line command into a single line one. 

More often than not, you will rewrite some code and it will now be short enough to fit on one line but Black would not reformat it onto one line if this option isn't set to true. So you end up with random multi-line commands that don't need to be multi-line.

Reply:
I removed this because initially I thought that black's configuration did not have this. Turns out that the setting name is wrong: it should be `skip-magic-trailing-comma`.

Most likely it was not doing anything before. I've updated it.

![image](https://github.com/calliope-project/calliope-pathways/assets/72193617/b6b94809-7e1b-41e6-9951-8a2d0b8601ab)